.github/workflows/release-drafter.yml

```diff
@@ -3,10 +3,17 @@ on:
   push:
     branches:
       - master
+
+permissions:
+  contents: read
+
 jobs:
   update_release_draft:
     runs-on: ubuntu-latest
+    permissions:
+      contents: write  # for release-drafter/release-drafter to create a github release
+      pull-requests: write  # for release-drafter/release-drafter to add label to PR
```

Comment:
We don't use such feature ... so this permission is not needed.

Reply:
That makes sense. Do you prefer that I comment/remove just the permissions or can I comment up the whole action? It would make it clearer that this feature is now unused and would prevent it to run (and fail) on every push.

Reply:
You can simply remove this permission. 

We only execute action on push event to master branch.
We don't have configuration for `autolabeler`

Reply:
Ooh sorry the misunderstanding. Got it now, will do